frontend/src/lib/components/Projects/ProjectTable.svelte

```diff
@@ -118,9 +127,27 @@
               </span>
             </td>
           {/if}
+          {#if canManage}
+            <td class="p-0">
+              <Dropdown>
```

Comment:
rather than adding this code here it would be better to take advantage of the `actions` slot. Take a look at how the ProjectTable is used in `AdminProjects.svelte`